packages/@aws-cdk/region-info/test/__snapshots__/region-info.test.js.snap

```diff
@@ -914,7 +914,7 @@ Object {
       "application-autoscaling": "application-autoscaling.amazonaws.com",
       "autoscaling": "autoscaling.amazonaws.com",
       "codedeploy": "codedeploy.amazonaws.com",
-      "ec2": "ec2.sc2s.sgov.gov",
+      "ec2": "ec2.amazonaws.com",
```

Comment:
I'm not certain about this one given that it's a different service principal than the others. @rix0rrr, humbly asking for your review and eyes next week because I'm not certain what I'm doing is right.

Reply:
Update: docs say it is still `ec2.sc2s.sgov.gov` but they could be wrong, tracking internally.